modules/services/gpg-agent.nix

```diff
@@ -246,8 +256,7 @@ in {
           ++ optional (cfg.maxCacheTtlSsh != null)
           "max-cache-ttl-ssh ${toString cfg.maxCacheTtlSsh}"
           ++ optional (cfg.pinentryFlavor != null)
-          "pinentry-program ${pkgs.pinentry.${cfg.pinentryFlavor}}/bin/pinentry"
-          ++ [ cfg.extraConfig ]);
+          "pinentry-program ${pinentryFlavor}" ++ [ cfg.extraConfig ]);
```

Comment:
Kind of annoyed at the list being inlined here, but the format script wouldn't be forced into putting a newline here...

Reply:
The standard trick for forcing newlines used elsewhere is to put `# \` at the end of the line. For example,

``` nix
          ++ optional (cfg.pinentryFlavor != null)
          "pinentry-program ${pinentryFlavor}" # \
          ++ [ cfg.extraConfig ]);
```

It's not the most pretty but I think it's not too bad.